### PR TITLE
Feature/vocabularies

### PR DESCRIPTION
**Pull request by PalomoIFCA**
Added test i1_01m: tests if the metadata follows contrdled vocabularies and i2_01m which is about the documentation and identification of those parameters
This helps with issue #92 and  issue #181

This will be upgraded when more metadata elements are included in the metadata and more vocabularies are implemented

### CODE REVIEW

File: api/utils.py [hunk 1]
```diff
@@ -867,3 +867,28 @@ def check_fairsharing_abbreviation(fairlist, abreviation):
...
             return (100, "Your metadata standard appears in Fairsharing")
     return (0, "Your metadata standard has not been found in Fairsharing")
+
+
+def check_ORCID(orcid):
+    headers = {
+        "Accept": "application/xml",
+    }
+
+    response = requests.get(orcid, headers=headers)
```
Comment (orviz) [inline api/utils.py:877]: ORCID is not a URL, what is this method expecting as input argument?

Besides the FAIR EVA code already imports [`idutils` library which has a `is_orcid()` method](https://idutils.readthedocs.io/en/latest/); it is better to use this one

File: plugins/epos/config.ini [hunk 2]
```diff
@@ -142,7 +142,20 @@ terms_data_model = []
 #metadata standard
 metadata_standard = ['DCAT']
 
-
+#terms that use vocabularies and vocabualries used
+vocabularies=['ROR','PIC','imtypes','TRL','spatial','temporal','Rolecode','spdx','ORCID']
+
+vocabularies_identifiers =['https://ror.org/','https://ec.europa.eu/info/funding-tenders/opportunities/portal/screen/how-to-participate/participant-register','https://www.iana.org/assignments/media-types/media-types.xhtml','TRL','spatial','https://www.iso.org/iso-8601-date-and-time-format.html','Rolecode','https://spdx.org/licenses/','https://orcid.org/']
```
Comment (orviz) [inline plugins/epos/config.ini:148]: shouldn't be better using a mapping here, like a dict in Python or JSON? is "spatial" a vocabulary?

File: plugins/epos/plugin.py [hunk 3]
```diff
@@ -1025,6 +1032,98 @@ def rda_a1_1_01d(self):
...
+        This indicator is linked to the following principle: I1: (Meta)data use a formal,
+        accessible, shared, and broadly applicable language for knowledge representation.
+
+        The indicator serves to determine that an appropriate standard is used to express
+        knowledge, in particular the data model and format.
+
+        Returns
+        -------
+        points
+            100/100 If the file format is listed under IANA Internet Media Types
```
Comment (orviz) [inline plugins/epos/plugin.py:1048]: Not only media types is being checked here

**Review by orviz — CHANGES_REQUESTED** [review]

File: plugins/epos/plugin.py (hunk 3 above)
Comment (PalomoIFCA) [reply plugins/epos/plugin.py]: Corrected

File: api/utils.py (hunk 1 above)
Comment (PalomoIFCA) [reply api/utils.py:877]: ORCID is sometimes expressed as a URL. Now it can support both cases

Comment (PalomoIFCA) [reply api/utils.py:877]: Sadly for the time being `idutils.is_orcid()` does not work with some  URLS like https://orcid.org/0000-0001-5750-0872/Project_leader but it does work with https://orcid.org/0009-0000-2496-4136 so i am not sure on what to do

Comment (PalomoIFCA) [reply api/utils.py]: Solved, check_orcid() is removed, cases not recognised by is_orcid will not ve accepted

File: plugins/epos/config.ini (hunk 2 above)
Comment (orviz) [reply plugins/epos/config.ini]: agree, at least for the time being

**Review by orviz — APPROVED** [review]

**Pull request merged**